packages/core/src/Engine.ts

```diff
@@ -216,7 +219,12 @@ export class Engine extends EventDispatcher {
     magentaTextureCube.setPixelBuffer(TextureCubeFace.NegativeZ, magentaPixel);
     magentaTextureCube.isGCIgnored = true;
 
-    const depthTexture2D = new Texture2D(this, 1, 1, TextureFormat.Depth16, false);
+    let depthFormat = TextureFormat.Depth16;
+    if (!hardwareRenderer.canIUse(GLCapabilityType.depthTexture)) {
+      depthFormat = TextureFormat.R16G16B16A16;
```

Comment:
Maybe do not need to fallback `TextureFormat.R16G16B16A16`,Engine will use `this._depthTexture2D ` only shader use sampler `SAMPLER_2D_SHADOW`.
If not support, `this._depthTexture2D` undefined is OK！